src/dune_pkg/opamUrl0.ml

```diff
@@ -30,10 +30,16 @@ let is_version_control t =
 
 let is_local t = String.equal t.transport "file"
 
-let local_or_git_only url loc =
+let is_tarball t =
```

Comment:
Should this not support zip files as well? From some cursory checking I see opam-repository has about 200 ZIP files and it would be good if we supported that.

Reply:
I will stick with your top-level suggestion and delay zip for latter. Indeed, contrary to tar it seems to be the code to support `zip` is currently missing.